<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotWindow</class>
 <widget class="QMainWindow" name="PlotWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCustomPlot" name="customPlot" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProperties">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_Plot"/>
   </widget>
   <widget class="QMenu" name="menuAppearance">
    <property name="title">
     <string>Appearance</string>
    </property>
    <addaction name="actionConnect_Points"/>
    <addaction name="actionFrame_Rate"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionData_Points"/>
    <addaction name="actionClear"/>
   </widget>
   <addaction name="menuProperties"/>
   <addaction name="menuAppearance"/>
   <addaction name="menuData"/>
  </widget>
  <action name="actionData_Points">
   <property name="text">
    <string>Data Points</string>
   </property>
  </action>
  <action name="actionConnect_Points">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Connect Points</string>
   </property>
  </action>
  <action name="actionSave_Plot">
   <property name="text">
    <string>Save Plot</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionFrame_Rate">
   <property name="text">
    <string>Frame Rate</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
